test/test_prior.py

```diff
@@ -13,20 +13,21 @@
 from pypesto.objective import NegLogParameterPriors
 from pypesto.objective.priors import get_parameter_prior_dict
 
+scales = ['lin', 'log', 'log10']
```

Comment:
This would make the pypesto API dependent on `petab`, which I would rather avoid. It should be possible to use these functions without petab and we don't want that functionality to break just because petab changed some of its literals.

Reply:
Then it might be worthwhile for pyPESTO to maintain its own set of constants (no need now).

Reply:
agreed!